### PR TITLE
✨ NEW: Add aiida tutorial 2022 event

### PR DESCRIPTION
**Pull request by chrisjsewell**

### CODE REVIEW

File: docs/news/posts/2022-08-23-aiida-tutorial-event.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Key Details
+
+### Online demonstration
+
+**When:** a 2-hour session on October 4th 2022, with two options to accommodate people from different time zones: 8:00-10:00 GMT and 15:00-17:00 GMT.
+
+**Where:** Virtual Zoom Meeting, details will be communicated on October 1st.
+
+**Registration:** Registration is free of charge, simply fill in the following (short) form before October 1st 2022:
```
Comment (mbercx) [inline docs/news/posts/2022-08-23-aiida-tutorial-event.md:31]: Maybe also put "Registration is closed!" here.

File: docs/news/posts/2022-08-23-aiida-tutorial-event.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+**Where:** Talks will be pre-recorded and made available to participants before the event. Hands-on tutorials will be held _via_ Zoom, with participants running the tutorial in their browser by accessing a multi-user AiiDAlab deployed on the [Azure Kubernetes Service](https://azure.microsoft.com/en-us/products/kubernetes-service/).
+
+**Registration**: Registration is closed!
+
+## Organisers
+
+The tutorial is organised by Chris Sewell (EPFL, CH), Marnik Bercx (EPFL, CH) and Giovanni Pizzi (EPFL, CH).
+
+Tutorial lectures and assistance during hands-on sessions will be provided by the organisers and a team of core AiiDA developers: Jusong Yu, Francisco Ramirez, Sebastiaan Huber, Flaviano dos Santos, and Carl Simon Adorf.
```
Comment (mbercx) [inline docs/news/posts/2022-08-23-aiida-tutorial-event.md:49]: Can you also still add "Aliaksandr Yakutovich" here?

**Review by mbercx — APPROVED** [review]
Thanks @chrisjsewell! Just a few small comments, and perhaps we can also add the event to the past events list, e.g.:

[4-7 Oct 2022] Online tutorial on running and writing workflows with AiiDA – available for all timezones ~ 80 participants

File: docs/news/posts/2022-08-23-aiida-tutorial-event.md (hunk 2 above)
Comment (chrisjsewell) [inline docs/news/posts/2022-08-23-aiida-tutorial-event.md:49]:
```suggestion
Tutorial lectures and assistance during hands-on sessions will be provided by the organisers and a team of core AiiDA developers: Jusong Yu, Francisco Ramirez, Sebastiaan Huber, Flaviano dos Santos, Carl Simon Adorf, and Aliaksandr Yakutovich.
```

File: docs/news/posts/2022-08-23-aiida-tutorial-event.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### Online demonstration
+
+**When:** a 2-hour session on October 4th 2022, with two options to accommodate people from different time zones: 8:00-10:00 GMT and 15:00-17:00 GMT.
+
+**Where:** Virtual Zoom Meeting, details will be communicated on October 1st.
+
+**Registration:** Registration is free of charge, simply fill in the following (short) form before October 1st 2022:
+
+[https://forms.gle/MjXi8Fb1rXquyETb8](https://forms.gle/MjXi8Fb1rXquyETb8)
```
Comment (chrisjsewell) [inline docs/news/posts/2022-08-23-aiida-tutorial-event.md:33]:
```suggestion
**Registration closed on October 1st 2022**
```

**Pull request merged**